app/javascript/selectors/answersSelector.js

```diff
@@ -1,10 +1,44 @@
 import {
   getAllActiveQuestions,
   getAllActiveQuestionsForCategory,
-  getAllQuestionsByCategoryId
+  getAllActiveQuestionIdsForCategory,
 } from "./questionsSelector";
 import { getCategoriesForBuilding } from "./categoriesSelector";
 
+export function getAnswersForBuilding(buildingId, state) {
+  let answers = state.buildings[buildingId].answers;
+  answers = Object.keys(answers).filter((id) => {
+    let email = answers[id].delegation_email;
+    let text = answers[id].text;
+    return !email && !text;
+  })
+  .reduce((newAnswers, id) => {
+    newAnswers[id] = answers[id];
+    return newAnswers
+  }, {});
+  return answers
+}
+
+export function getAnswersForCategoryAndBuilding(categoryId, buildingId, state) {
+  const buildingQuestions = state.buildings[buildingId].questions;
+  let categoryQuestions = state.categories[categoryId].questions;
+  categoryQuestions = buildingQuestions.filter((id) => {
+    return categoryQuestions.includes(Number(id));
```

Comment:
small nitpick but I think `parseInt` is a bit cleaner/safer than `Number`

https://stackoverflow.com/questions/4564158/what-is-the-difference-between-parseintstring-and-numberstring-in-javascript